boskos/metrics/metrics.go

```diff
@@ -39,33 +40,35 @@ var (
 	promMetrics = prometheusMetrics{
 		BoskosState: map[string]map[string]prometheus.Gauge{},
 	}
-	resources common.ResTypes
+	resources, states common.ResTypes
```

Comment:
hummmmm, states is not really a common.ResTypes?

Reply:
Should change restype to be a something like StringArrayFlag ?